bundles/binding/org.openhab.binding.km200/src/main/java/org/openhab/binding/km200/internal/KM200Comm.java

```diff
@@ -175,6 +183,9 @@ public static String bytesToHex(byte[] data) {
             } catch (HttpException e) {
                 logger.error("Fatal protocol violation: {}", e);
```

Comment:
If the last argument is a `Throwable` as in this case, the stacktrace is logged but the `{}` will be printed.  If you only want the exception method in one line, use `e.getMessage()` instead of `e` and the message will substitute the `{}` in the message.